### ISSUE TITLE
Fix hover text color on apply date facet button

### ISSUE THREAD

**Issue by kamerynB** [project-lux/lux-frontend#181]
**Problem Description:**
The `Apply` button on date facets turns black on hover which does not pass color contrast testing.

**Expected Behavior/Solution:**
Change the text color on hover to a contrast approved color. The background color will become white and the text color will become the dark blue.

**Requirements:**

- [x] hover css property should be the following:
```
  &:hover {
    background-color: ${theme.color.white};
    color: ${theme.color.primary.darkBlue};
    border-color: ${theme.color.primary.darkBlue};
  }
```

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [x] Wireframe/Mockup - Mike
- [ ] ~Committee discussions - Sarah~
- [ ] ~Feasibility/Team discussion - Sarah~
- [ ] ~Backend requirements - N/A~
- [x] Frontend requirements - Kam
- [ ] ~Questions~

**UAT/LUX Examples:**

- https://lux-front-tst.collections.yale.edu/view/results/objects/?q=%7B%22AND%22%3A%5B%7B%22text%22%3A%22andy%22%2C%22_lang%22%3A%22en%22%7D%2C%7B%22text%22%3A%22warhol%22%2C%22_lang%22%3A%22en%22%7D%5D%7D&sq=andy+warhol
    - Click on the Creation Date facet and hover the mouse over the "Apply" button
- https://lux.collections.yale.edu/view/results/objects/?q=%7B%22AND%22%3A%5B%7B%22text%22%3A%22andy%22%2C%22_lang%22%3A%22en%22%7D%2C%7B%22text%22%3A%22warhol%22%2C%22_lang%22%3A%22en%22%7D%5D%7D&sq=andy+warhol (prod) 

**Dependencies/Blocks:**
N/A

**Related Github Issues:**
N/A

**Related links:**

- Bugherd: https://www.bugherd.com/projects/284041/tasks/2115

**Wireframe/Mockup:**
![Screenshot 2024-04-10 at 12 18 58 PM](https://github.com/project-lux/lux-frontend/assets/140102078/da19960e-1b01-4206-b6ca-c85f76937dd8)

**Comment by roamye** [comment]
@kamerynB Is the link in the UAT links working for you? It tells me the site can't be reached(I tried on and off VPN).
Maybe we could use this link instead: https://lux-front-tst.collections.yale.edu/view/results/objects/?q=%7B%22AND%22%3A%5B%7B%22text%22%3A%22andy%22%2C%22_lang%22%3A%22en%22%7D%2C%7B%22text%22%3A%22warhol%22%2C%22_lang%22%3A%22en%22%7D%5D%7D&sq=andy+warhol

**Comment by kamerynB** [comment]
@roamye Whoops I added my localhost URL. I'll update it.

**Comment by roamye** [comment]
looks good in prod, closing
